_includes/partials/individual-component.html

```diff
@@ -45,7 +45,54 @@ <h3>Compatibility:</h3>
 </script>
 
 <script>
-  function handleClick(link){
-    window.open(link);
+  function toggleFullScreenImage(patternId) {
+    var fullScreenContainer = document.getElementById(`fullScreenContainer${patternId}`);
+
+    if (!fullScreenContainer) {
+      createFullScreenContainer(patternId);
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.removeChild(fullScreenContainer);
+      document.body.style.overflow = 'auto';
+    }
   }
-</script>
\ No newline at end of file
+
+  function createFullScreenContainer(patternId) {
```

Comment:
please write this javascript to specific to any .js file otherwise in a long term people will keep gonna add `js` and` css` and `html` in same files and will hard to manage. And this functionality can also be used in any other place if we need to some thing in more broad view